src/main/resources/mailtemplates/de/topic-deleted.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title th:text="'Kafka-Topic ' + ${topic_name_abbreviated} + ' wurde gelöscht'">Kafka-Topic wurde
+        gelöscht</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<!--/*@thymesVar id="user_name" type="java.lang.String"*/-->
+<!--/*@thymesVar id="topic_name" type="java.lang.String"*/-->
+<!--/*@thymesVar id="galapagos_topic_url" type="java.lang.String"*/-->
+<!--/*@thymesVar id="environment_name" type="java.lang.String"*/-->
+<body style="font-family: Helvetica, Arial, sans-serif; font-size: 11pt">
+<p>
+    Hallo lieber Galapagos-Benutzer,
+</p>
+<p>Benutzer <b><i>
```

Comment:
The whole text should be: "Hallo lieber Galapagos-Benutzer, das interne Topic <topicname> wurde automatisch als gelöscht erkannt, da es auf dem Kafka-Cluster <env> nicht mehr zu existieren scheint. Die zugehörigen Metadaten wurden aus Galapagos entfernt. Es wird empfohlen, über Galapagos angelegte interne Topics auch ausschließlich über Galapagos zu löschen. Außerhalb von Galapagos angelegte interne Topics können natürlich jederzeit gelöscht werden."